aleph-client/src/pallets/utility.rs

```diff
@@ -1,7 +1,9 @@
 use crate::{api, BlockHash, Call, SignedConnection, TxStatus};
 
+/// Any object that implements pallet utility api.
```

Comment:
IMHO you shouldn't repeat everytime that a trait represents some set of objects, just focus on describing what that trait is doing.

Reply:
done